src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherForm.java

```diff
@@ -38,6 +49,26 @@ public String getTripID() {
 		return StringUtils.EMPTY;
 	}
 	
+	public boolean isCancelPDPPaymentsActionAvailable() {
+	    if (getRecurringDisbursementVoucherPaymentMaintenanceService().hasCancelPermission(GlobalVariables.getUserSession().getPerson())) {
+    	    Collection<PaymentDetail> details = getRecurringDisbursementVoucherDocumentService().findPaymentDetailsFromRecurringDisbursementVoucher((RecurringDisbursementVoucherDocument)getDocument());
+    	    for (PaymentDetail detail : details) {
+    	        String paymentDetailStatus = detail.getPaymentGroup().getPaymentStatusCode();
+    	        Person person = GlobalVariables.getUserSession().getPerson();
+    	        if (paymentDetailStatus != null && 
+    	                getRecurringDisbursementVoucherDocumentService().isPaymentCancelable(person, paymentDetailStatus)) {
```

Comment:
For consistency with our other code, I would suggest moving the "&&" to the second line of the "if" statement.